src/Components/test/E2ETest/Tests/ComponentRenderingTest.cs

```diff
@@ -408,7 +408,12 @@ public void CanUseJsInteropToReferenceElements()
         [Fact]
         public void CanUseFocusExtensionToFocusElement()
         {
+            Browser.Manage().Window.Size = new System.Drawing.Size(100, 200);
```

Comment:
@SteveSandersonMS thoughts on testing this?

Reply:
Assuming this logic works, it looks good to me.